Cannot run "push" command from the developers documentation

Following the instructions here:

https://github.com/Azure/brigade/blob/master/docs/topics/developers.md#minikube-configuration

It says:
> Running `VERSION=latest make docker-build docker-push` will push the Brigade images to the Minikube Docker daemon.

But this doesn't work for me. I get a permission denied error because it's trying to publish out to docker.io.

```console
> VERSION=latest make docker-build docker-push
docker push deis/brigade-api:latest
The push refers to a repository [docker.io/deis/brigade-api]
ba7576f5d121: Preparing
b7c3763077f3: Preparing
cd7100a72410: Preparing
denied: requested access to the resource is denied
make: *** [brigade-api-push] Error 1
```

A few thoughts:

* The list of addons in that document state that the minikube addon `registry` should be disabled, but I think it needs to be enabled for this to work
* The makefile, I believe, needs to point at the local registry, instead of docker.io? I'm not super-familiar with the minikube registry though.

--- a/docs/topics/developers.md
+++ b/docs/topics/developers.md
@@ -88,8 +88,9 @@ Docker daemon:
 $ eval $(minikube docker-env)
 ```
 
-Running `VERSION=latest make docker-build docker-push` will push the Brigade images to the Minikube Docker
-daemon.
+Running `VERSION=latest make docker-build` will push the Brigade images to the Minikube Docker
+daemon. You can verify this by running `docker images`. You should see the `latest` tags for
+the brigade images.
 
 Now create a custom `values.yaml` file for the chart, and set the images to all
 pull the `latest` image:

